forms-shared/src/summary-email/SummaryEmail.tsx

```diff
@@ -80,8 +80,8 @@ const StringValueRenderer = ({ value, isLast }: SummaryStringValueRendererProps)
 )
 
 const FileValueRenderer = ({ fileInfo, isLast }: SummaryFileValueRendererProps) => {
-  const fileIdUrlMap = useFileIdUrlMap()
-  const fileUrl = fileIdUrlMap[fileInfo.id].url
+  const fileIdInfoMap = useFileIdInfoMap()
+  const fileUrl = fileIdInfoMap[fileInfo.id].url
```

Comment:
Must be `fileIdInfoMap[fileInfo.id]?.url` or it will fail catastrophically in edge cases.

Reply:
FYI: https://www.typescriptlang.org/tsconfig/#noUncheckedIndexedAccess